packages/app/cypress/e2e/cypress-in-cypress-e2e.cy.ts

```diff
@@ -148,7 +148,7 @@ describe('Cypress In Cypress E2E', { viewportWidth: 1500, defaultCommandTimeout:
   })
 
   it('shows a compilation error with a malformed spec', { viewportHeight: 596, viewportWidth: 1000 }, () => {
-    const expectedAutHeight = 500 // based on explicitly setting viewport in this test to 596
+    const expectedAutHeight = 456 // based on explicitly setting viewport in this test to 596
```

Comment:
Change `<a>` to `<input>` for the Studio URL input - this caused the breakpoint for `flex: flex-wrap` to change slightly. Seems like it's not a big deal.